src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    public static String getColour() {
+        int random = (int) (Math.random() * (5 - 1)) + 1;
```

Comment:
What is 5 - 1? Let's create constant

Reply:
It is formula that is widely use to generate numbers in range from max to min.
Math.random() * (max - min)) + min
you want me to make method for that? Or should i use java.util.concurrent.ThreadLocalRandom